src/ValidationError.js

```diff
@@ -2,7 +2,13 @@
 let strReg = /\$\{\s*(\w+)\s*\}/g;
 
 let replace = str =>
-  params => str.replace(strReg, (_, key) => params[key] || '')
+  params => str.replace(strReg, (_, key) => {
+    const val = params[key]
+    if (typeof val === 'number') {
+      return val
+    }
+    return val || ''
```

Comment:
how about just `String(val)`? that would print `"null"` and `"undefined"` but that seems like the right behavior anyway here?